src/NuGet.Core/NuGet.Protocol/Plugins/Messages/GetAuthenticationCredentialsResponse.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Plugins
+{
+    public class GetAuthenticationCredentialsResponse
+    {
+        public string Username { get; }
+
+        public string Password { get; }
+
+        public string Message { get; }
+
+        /// <summary>
+        /// Gets or sets the list of authentication types this credential is applicable to. Useful values include
+        /// <c>basic</c>, <c>digest</c>, <c>negotiate</c>, and <c>ntlm</c>
+        /// </summary>
+        public IList<string> AuthTypes { get; }
```

Comment:
Add XML doc comments to the class and all its public members or do not add any at all.